src/main/java/org/fundacion/jala/converter/controller/ProjectController.java

```diff
@@ -32,7 +41,7 @@
      * Creates a project in data base.
      *
      * @param projectName is a String with the project's name.
-     * @param userId is an integer with the user's id.
+     * @param userId      is an integer with the user's id.
```

Comment:
There are extra spaces between "userId" and "is".

Reply:
It is done.